BackendApartmentReservation/BackendApartmentReservation/Controllers/ApartmentsController.cs

```diff
@@ -9,11 +11,19 @@ namespace BackendApartmentReservation.Controllers
     [ApiController]
     public class ApartmentsController : ControllerBase
     {
+        private ICustomLogger<ApartmentsController> _logger;

```

Comment:
## Question
Possible to make it readonly? -Sonar suggests

Reply:
Definitely :)